test/libsolidity/syntaxTests/inheritance/override/modifier_inherited_different_signature.sol

```diff
@@ -0,0 +1,10 @@
+contract A {
+    modifier f(uint a) virtual { _; }
+}
+contract B {
+    modifier f() virtual { _; }
+}
+contract C is A, B {
+}
+// ----
+// TypeError: (100-122): Derived contract must override modifier "f". Two or more base classes define modifier with same name and parameter types.
```

Comment:
The error message isn't precise in this case since the base modifiers don't have the same parameter types.